<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterMoreInforForm</class>
 <widget class="QWidget" name="RegisterMoreInforForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="ImageLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(25, 255, 255);</string>
    </property>
    <property name="text">
     <string>拖动一张图片到
这里或者点击此
处选择一张图片,
作为你的头像</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNickName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>19</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>昵称:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>电子邮件:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPhone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>   手机号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAddress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>   所在地:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPhone">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMail">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNickName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelYear">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>58</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生肖:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConstellation">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>58</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>星座:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelYear2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>58</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelConstellation2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>58</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBlood">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>血型:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxBlood">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB型</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonPasswdProtect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>83</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border-image: url(:/PushButton.png);
}
QPushButton:hover
{
	border-image: url(:/PushButtonHover.png);
}
QPushButton:pressed
{
	border-image: url(:/PushButtonPressed.png);
}</string>
    </property>
    <property name="text">
     <string>密码保护</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>83</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border-image: url(:/PushButton.png);
}
QPushButton:hover
{
	border-image: url(:/PushButtonHover.png);
}
QPushButton:pressed
{
	border-image: url(:/PushButtonPressed.png);
}</string>
    </property>
    <property name="text">
     <string>好</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxProvince">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>ImageLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditNickName</tabstop>
  <tabstop>comboBoxBlood</tabstop>
  <tabstop>comboBoxProvince</tabstop>
  <tabstop>comboBoxCity</tabstop>
  <tabstop>lineEditPhone</tabstop>
  <tabstop>lineEditMail</tabstop>
  <tabstop>pushButtonPasswdProtect</tabstop>
  <tabstop>pushButtonOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
